### PR TITLE
[TAAS-35] Configuration file documentation.

### PR DESCRIPTION
**Pull request by pjf**
I've put together documentation on our config file format, but I'm not sure how well it holds together, and it almost certainly needs editing.

@amyvbenson, could you be so kind as to read over them? Please feel free to correct/amend/suggest/question/criticise liberally.

Many, many thanks!

~ Paul

### CODE REVIEW

File: docs/Config.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+You can see our existing configurations [in this repo](../taas/config.d).
+
+## Top level format
+
+Our configuration is in [YAML](http://yaml.org/) format, and consists of a sequence of sources (spreadsheets we're reading from) along with a description how that this data should be transformed into JSON.
+
+Each source can contain a number of exports, which would usually be version (`v1`, `v2`, etc), allowing for us to use the same data to generate different outputs. These directly translate to filenames on output.
+
+Each version has a human-friendly spreadsheet `url` (which we use to generate our own URL for data extraction in CSV), and a `mapping`, which describes how we map spreadsheet columns into JSON structions.
```
Comment (amyvbenson) [inline docs/Config.md]: @pjf what's a struction?

**Review by amyvbenson — COMMENTED** [review]
Other than that all makes sense :)

File: docs/Config.md (hunk 1 above)
Comment (pjf) [reply docs/Config.md]: @amyvbenson : It's a cross between a structure and a function, such as [JSONP](https://en.wikipedia.org/wiki/JSONP).

Alas, we don't implement these yet, so I really should correct the docs. :)

Comment (pjf) [reply docs/Config.md]: Fixed! :)

**Pull request merged**